docs/docs/howto/logging.md

```diff
@@ -44,7 +44,7 @@ following variables to your container definition.
 ```yaml
 spec: # v1.PodSpec
   containers:
-    image: "ghcr.io/edgelesssys/contrast/coordinator:latest"
+    image: "ghcr.io/edgelesssys/contrast/coordinator:latest@sha256:dbcc56f4d19882b939f461a71a005cb442e963dc31d59f2e1195d54366575e20"
```

Comment:
This change is wrong. Images here mustn't be pinned, they are replaced on release!